substrate/client/rpc-spec-v2/src/chain_head/chain_head_follow.rs

```diff
@@ -67,6 +73,8 @@ pub struct ChainHeadFollower<BE: Backend<Block>, Block: BlockT, Client> {
 	sub_id: String,
 	/// The best reported block by this subscription.
 	best_block_cache: Option<Block::Hash>,
```

Comment:
Nit: when I read `best_block_cache` below I thoguht it was a cache of best blocks. `current_best_block` would be clearer to me :)

Reply:
That makes sense, thanks!